src/Storages/StorageReplicatedMergeTree.h

```diff
@@ -115,6 +115,10 @@ class StorageReplicatedMergeTree final : public ext::shared_ptr_helper<StorageRe
       */
     void drop() override;
 
+    /** Removes a specific replica from Zookeeper. If replica is local, it works same as `drop` method.
```

Comment:
Other code and comments tell it is impossible to drop local replica. Incorrect comment?

Reply:
Yes. I will fix the comment, the local replica can only drop by `DROP TABLE`